nre/bpf/nodefw/Cargo.toml

```diff
@@ -0,0 +1,3 @@
+[workspace]
+resolver = "2"
+members = ["xtask", "nodefw", "nodefw-common"]
```

Comment:
Can we eliminate this workspace and move these crates to the main /crates directory?

If we need them to be exempt from hakari we can easily do that

Reply:
this might break the xtask alias, but i'll try